lib/kits/events/tracker.js

```diff
@@ -20,19 +20,28 @@ const createTrackerClient = () => {
 };
 
 function getPool() {
-    let data = clientLocalStorage.get('event-tracker-pool');
-    if (Array.isArray(data) === false) {
-        data = [];
-    }
-    data = data.filter((evt) => typeof evt._i === 'string');
+    const data = clientLocalStorage.get('event-tracker-pool');
 
-    return data;
+    return Array.isArray(data)
+        ? data.filter((evt) => typeof evt._i === 'string')
+        : [];
 }
 
-let pool = getPool();
+const unloadHandler = () =>
+    clientLocalStorage.set('event-tracker-pool', pool.concat(getPool()));
 
+let pool;
```

Comment:
The `pool` is however still a singleton. One issue I can think of there is if an integrator were to be using multiple `eventsTrackUrl`s, which doesn't seem very likely but stranger things have happened...

The extreme version of this PR would maybe have each Tracker have its own `pool` instance property